packages/plugins/persisted-operations/src/index.ts

```diff
@@ -42,16 +44,50 @@ export interface UsePersistedOperationsOptions {
    * The path to the persisted operation id
    */
   extractPersistedOperationId?: ExtractPersistedOperationId
-}
+
+  /**
+   * Whether to skip validation of the persisted operation
+   */
+  skipValidation?: boolean
+} & (
+  | {
+      /**
+       * A function that fetches the persisted operation as AST
+       */
+      getPersistedOperation(key: string): PromiseOrValue<DocumentNode | null>
+
+      operationType: PersistedOperationType.AST
+    }
+  | {
+      /**
+       * A function that fetches the persisted operation as AST
+       */
+      getPersistedOperation(key: string): PromiseOrValue<string | null>
+
+      /**
+       * The type of the persisted operation
+       *
+       * AST: The persisted operation is an AST
+       * SDL: The persisted operation is an SDL
+       *
+       * @default PersistedOperationType.SDL
+       */
+      operationType?: PersistedOperationType.SDL
+    }
+)
 
 export function usePersistedOperations<
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   TPluginContext extends Record<string, any>,
 >({
-  getPersistedOperation,
   allowArbitraryOperations = false,
   extractPersistedOperationId = defaultExtractPersistedOperationId,
+  getPersistedOperation,
+  operationType,
+  skipValidation = true,
```

Comment:
default value should be `false`. Maybe also rename it to `skipDocumentValidation` to me more explicit.

Reply:
Done 👍

Reply:
You also need to update the docs to reflect this